frontend/src/configs.js

```diff
@@ -1,17 +1,34 @@
-const URI_USER_SVC = process.env.URI_USER_SVC || 'http://localhost:8000';
-const URI_MATCHING_SVC = process.env.URI_MATCHING_SVC || 'http://localhost:8001';
-const URI_COMM_SVC = process.env.URI_COMM_SVC || 'http://localhost:8002';
-const URI_COLLAB_SVC = process.env.URI_COLLAB_SVC || 'http://localhost:8003';
+// Reverse proxy
+const URI_SVC = process.env.URI_SVC || 'http://localhost:8080';
 
-const PREFIX_USER_SVC = '/api/user';
-const PREFIX_COMM_SVC = '/api/chat';
+// User service
+const PREFIX_USER_SVC = '/api/user-service';
+const URL_USER_SVC = URI_SVC + PREFIX_USER_SVC;
 
-export const URL_USER_SVC = URI_USER_SVC + PREFIX_USER_SVC;
-export const URL_MATCHING_SVC = URI_MATCHING_SVC;
-export const URL_COMM_SVC = URI_COMM_SVC + PREFIX_COMM_SVC;
-export const URL_COLLAB_SVC = URI_COLLAB_SVC;
+const PREFIX_USER_SVC_SIGNIN = '/signin';
+const PREFIX_USER_SVC_SIGNUP = '/signup';
+// TODO: Change to NGINX URL to check for JWT validity in the future, using user service URL at the moment
+const PREFIX_USER_SVC_JWT_VERIFICATION = '/verify-jwt';
+const PREFIX_USER_SVC_LOGOUT = '/logout';
+const PREFIX_USER_SVC_UPDATE = '/update';
+const PREFIX_USER_SVC_DELETE = '/delete';
 
-const PREFIX_COMM_SVC_READ_CHAT = '/read';
+export const URL_USER_SVC_SIGNIN = URL_USER_SVC + PREFIX_USER_SVC_SIGNIN;
+export const URL_USER_SVC_SIGNUP = URL_USER_SVC + PREFIX_USER_SVC_SIGNUP;
+export const URL_USER_SVC_JWT_VERIFICATION = URL_USER_SVC + PREFIX_USER_SVC_JWT_VERIFICATION;
+export const URL_USER_SVC_LOGOUT = URL_USER_SVC + PREFIX_USER_SVC_LOGOUT;
+export const URL_USER_SVC_UPDATE = URL_USER_SVC + PREFIX_USER_SVC_UPDATE;
+export const URL_USER_SVC_DELETE = URL_USER_SVC + PREFIX_USER_SVC_DELETE;
+
+// Matching service
+export const URL_MATCHING_SVC_CONNECT = URI_SVC; // Socket
 
-export const URL_COMM_SVC_CONNECT = URI_COMM_SVC;
+// Communication service
+const PREFIX_COMM_SVC = '/api/communication-service';
+const URL_COMM_SVC = URI_SVC + PREFIX_COMM_SVC;
+const PREFIX_COMM_SVC_READ_CHAT = '/read';
 export const URL_COMM_SVC_READ_CHAT = URL_COMM_SVC + PREFIX_COMM_SVC_READ_CHAT;
+export const URL_COMM_SVC_CONNECT = URI_SVC; // Socket
+
+// Collaboration service
+export const URL_COLLAB_SVC_CONNECT = URI_SVC; // Socket

```

Comment:
I do not see the endpoints for question service, can you help to add here (if any) for consistency? Thanks!

Reply:
Sure, I can help to setup the base URL first, which can then be further extended in [this PR](https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g28/pull/50).